ArtemShchepets/src/main/java/week1/comparators/SellersSoldProductsComparator.java

```diff
@@ -11,4 +11,5 @@
     public int compare(Seller o1, Seller o2) {
```

Comment:
not valid compare) if they sold the same quantity of product?  Better by fullName